theseus_gui/src/helpers/theme.js

```diff
@@ -1,13 +1,17 @@
 /**
- * Sets the dark-mode or light-mode class on <html> if dark mode is on.
- * @param {Boolean} isDarkMode Bool value indicating if dark mode is on.
+ * Sets the html tag in the DOM to have the dark-mode class
  */
-export const toggleTheme = (isDarkMode) => {
-  if (isDarkMode) {
-    document.getElementsByTagName('html')[0].classList.remove('light-mode')
-    document.getElementsByTagName('html')[0].classList.add('dark-mode')
-  } else {
-    document.getElementsByTagName('html')[0].classList.remove('dark-mode')
-    document.getElementsByTagName('html')[0].classList.add('light-mode')
-  }
+export const setDarkTheme = () => {
+  document.getElementsByTagName('html')[0].classList.remove('light-mode')
+  document.getElementsByTagName('html')[0].classList.add('dark-mode')
 }
+
+/**
+ * Sets the html tag in the DOM to have the light-mode class
+ */
+export const setLightTheme = () => {
+  document.getElementsByTagName('html')[0].classList.remove('dark-mode')
+  document.getElementsByTagName('html')[0].classList.add('light-mode')
+}
```

Comment:
Not sure why these were split up. Ideally this func should just take a string which is a theme and just set a theme